src/pages/TermsOfService.ts

```diff
@@ -0,0 +1,148 @@
+import _ from 'lodash/fp';
+import { useState } from 'react';
+import { div, h, h1, img, WithKey } from 'react-hyperscript-helpers';
+import { ButtonOutline, ButtonPrimary, ButtonSecondary } from 'src/components/common';
+import { centeredSpinner } from 'src/components/icons';
+import { MarkdownViewer, newWindowLinkRenderer } from 'src/components/markdown';
+import scienceBackground from 'src/images/science-background.jpg';
+import { Ajax } from 'src/libs/ajax';
+import { signOut } from 'src/libs/auth';
+import colors from 'src/libs/colors';
+import { reportError, withErrorReporting } from 'src/libs/error';
+import * as Nav from 'src/libs/nav';
+import { useOnMount } from 'src/libs/react-utils';
+import { authStore } from 'src/libs/state';
+import * as Style from 'src/libs/style';
+import * as Utils from 'src/libs/utils';
+
+const TermsOfServicePage = () => {
+  const [busy, setBusy] = useState<boolean>();
+  const { isSignedIn, termsOfService } = authStore.get(); // can't change while viewing this without causing it to unmount, so doesn't need to subscribe
```

Comment:
Let's use `useStore` here instead of directly reading from the store.

```suggestion
  const { isSignedIn, termsOfService } = useStore(authStore);
```

Reply:
Also, this will have to be updated to use `signInStatus` instead of `isSignedIn` after #4254.